cli_tools/osconfig_agent/main.go

```diff
@@ -140,8 +140,8 @@ func run(ctx context.Context) {
 		} else {
 			setOsConfig(resp)
 			setPatchPolicies(resp.PatchPolicies)
-			runInventory()
 		}
+		runInventory()
```

Comment:
Curious about the reason behind this change

Reply:
The inventory run does not require a non error from the osconfig api. This is a bit of a hack to get around projects that don't have osconfig enabled but want to use inventory data. We will have a better solution in the future.